mono/metadata/image.c

```diff
@@ -543,6 +558,22 @@ load_metadata_ptrs (MonoImage *image, MonoCLIImageInfo *iinfo)
 			ptr += 4 - (pad % 4);
 	}
 
+	{
+		const char *p;
```

Comment:
Nit: can't say I'm a fan of the variable name here.

Reply:
`ptr` was already taken in this scope

Reply:
I'd prefer `heap_ptr` or `ptr2` since even in the relatively small block we have a second short name starting with p (pad), but it's not a big deal.